pyvista/plotting/plotter.py

```diff
@@ -3895,36 +3959,56 @@ def _add_legend_label(self, actor, label, scalars, color: Color):
     def add_volume(
         self,
         volume: pyvista.DataSet | pyvista.MultiBlock | NumpyArray[float],
-        scalars=None,
-        clim=None,
-        resolution=None,
-        opacity='linear',
-        n_colors=256,
-        cmap=None,
-        flip_scalars=False,
-        reset_camera=None,
-        name=None,
-        ambient=None,
-        categories=False,
-        culling=False,
-        multi_colors=False,
-        blending='composite',
-        mapper=None,
-        scalar_bar_args=None,
-        show_scalar_bar=None,
-        annotations=None,
-        pickable=True,
-        preference='point',
-        opacity_unit_distance=None,
-        shade=False,
-        diffuse=0.7,  # TODO: different default for volumes
-        specular=0.2,  # TODO: different default for volumes
-        specular_power=10.0,  # TODO: different default for volumes
-        render=True,
-        user_matrix=None,
-        log_scale=False,
+        scalars: str | NumpyArray[float] | None = None,
+        clim: float | tuple[float, float] | None = None,
+        resolution: VectorLike[float] | None = None,
+        opacity: Literal[
+            'linear',
+            'linear_r',
+            'geom',
+            'geom_r',
+            'sigmoid',
+            'sigmoid_1',
+            'sigmoid_2',
+            'sigmoid_3',
+            'sigmoid_4',
+            'sigmoid_5',
+            'sigmoid_6',
+            'sigmoid_7',
+            'sigmoid_8',
+            'sigmoid_9',
+            'sigmoid_10',
+            'sigmoid_15',
+            'sigmoid_20',
+            'foreground',
+        ]
+        | NumpyArray[float] = 'linear',
+        n_colors: int = 256,
+        cmap: str | list[str] | pyvista.LookupTable | None = None,
+        flip_scalars: bool = False,
+        reset_camera: bool | None = None,
+        name: str | None = None,
+        ambient: float | None = None,
+        categories: bool | int = False,
+        culling: Literal['front', 'back'] | bool = False,
```

Comment:
`frontface` and `backface` are also supported.
```suggestion
        culling: Literal['front', 'back', 'frontface', 'backface'] | bool = False,
```

Consider creating an alias since this option is re-used, e.g. with `add_volume`

```suggestion
        culling: `CullingOptions` | bool = False,
```